index.js

```diff
@@ -207,7 +207,7 @@ module.exports = function (str) {
   // U+00EC  0xEC  ì Ã¬  %C3 %AC
   .replace(/Ã¬/g, 'ì')
   // U+00ED  0xED  í Ã­  %C3 %AD
-  .replace(/Ã­\u00AD/g, 'í')
+  .replace(/Ã\u00AD/g, 'í')
```

Comment:
There was a nonvisible \uOOAD character in this line which prevented it from working.